docs/source/index.rst

```diff
@@ -68,6 +68,18 @@ You can subclass ``DocumentationView`` to require authentication and so add
 access control, and so on. You can customise the template to add other dynamic
 elements.
 
+Render a Global Table of Contents
+---------------------------------
+
+By default the ``toc`` block will only render the document's Table of Contents.
+
+If useful, a global Table of Contents can be rendered. This requires four steps:
+
+1. Add ``sphinx_view`` to your ``extensions`` in Sphinx's ``conf.py``
+2. Switch to the jsontoc builder with by using ``make jsontoc``
+3. Switch the ``json_build_dir`` to the new ``jsontoc`` directory
+4. Include the ``{{ doc.toctree|safe }}`` variable in your template
```

Comment:
I think we can merge these with the previous instructions.

Reply:
I think I found a more concise way to update the docs based on your suggestions. It's assuming we're having them use the new builder.